src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_adexchangebuyer.v1.4.json.baseline

```diff
@@ -4,24 +4,24 @@
 // 1. If not already done, enable the Ad Exchange Buyer API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/adexchangebuyer
-// 2. This sample uses Application Default Credentials for authentication.
-//    If not already done, install the gcloud CLI from
-//    https://cloud.google.com/sdk/ and run
-//    'gcloud beta auth application-default login'
-// 3. Install the Node.js client library and Application Default Credentials
+// 2. Install the Node.js client library and Application Default Credentials
```

Comment:
I did test logging.entries.list some time ago with this change and it worked fine, though I haven't tested the full range of possibilities.
I'm reasonably confident other configurations will work, but if you'd like me to be more comprehensive, let me know.